### PR TITLE
Fixes - l2discovery LAG support and unsubscribe event wrong host

### PR DESCRIPTION
**Pull request by edcdavid**
Includes the following fixes:
- https://github.com/test-network-function/l2discovery/pull/33
- https://github.com/test-network-function/l2discovery/pull/32
- https://github.com/test-network-function/l2discovery/pull/31
- https://github.com/test-network-function/privileged-daemonset/pull/52
- https://github.com/redhat-cne/ptp-listener-lib/commit/68d31fabc6ff9dd7323fb5b6ea93dc47b1ec9d2b

### CODE REVIEW

File: vendor/modules.txt [hunk 1]
```diff
@@ -593,7 +593,7 @@ k8s.io/apimachinery/third_party/forked/golang/reflect
 k8s.io/cli-runtime/pkg/genericclioptions
 k8s.io/cli-runtime/pkg/printers
 k8s.io/cli-runtime/pkg/resource
-# k8s.io/client-go v0.26.1 => k8s.io/client-go v0.25.4
+# k8s.io/client-go v0.26.3 => k8s.io/client-go v0.25.4
```
Comment (aneeshkp) [inline vendor/modules.txt:596]: why golcient version is  changed?

Comment (edcdavid) [reply vendor/modules.txt:596]: no specific requirement, just a regular update with go mod tidy

**Pull request merged**